dask/dataframe/rolling.py

```diff
@@ -110,10 +113,15 @@ def _call_method(self, method_name, *args, **kwargs):
             self.rolling_kwargs, method_name, args, kwargs)}
         if self.rolling_kwargs['axis'] in [0, 'rows']:
             # roll in the partition direction (will need to access neighbor)
+            window = self.rolling_kwargs['window']
+            tail_name = '{}-tailprev-{}'.format(old_name, window-1)
```

Comment:
`old_name` should already have some LONGHASHSTRING. Is it essential to add more?